test/src/e2e_vm_tests/test_programs/should_fail/private_function_fully_qualified/json_abi_oracle.json

```diff
@@ -0,0 +1 @@
+[]
```

Comment:
Actually this file can probably be removed?

Reply:
Good point, removed file.